### PR TITLE
Fix python plugin to work with new layer factory in CMake build

### PR DESCRIPTION
**Pull request by BlGene**

### CODE REVIEW

**Comment by reena96** [comment]
On running this command, I got the error below it. Kindly share your thoughts on how I can fix it.
python create_yolo_caffemodel.py -m deploy_vgg16_places365.prototxt -w yolo.weights -o yolo.caffemodel 

I0310 11:37:24.577980 14440 layer_factory.hpp:77] Creating layer relu1
F0310 11:37:24.578007 14440 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Relu (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)

**Comment by reena96** [comment]
Now, I am getting this error:
I0310 12:32:08.922411 16220 layer_factory.hpp:77] Creating layer data
F0310 12:32:08.922458 16220 db_lmdb.hpp:15] Check failed: mdb_status == 0 (2 vs. 0) No such file or directory
*** Check failure stack trace: ***
Aborted (core dumped)
